packages/rollup-plugin-html/demo/disable/index.html

```diff
@@ -0,0 +1,13 @@
+<script type="module" src="./src/my-app.js"></script>
```

Comment:
Wanted to demonstrate the the original behavior still works for "extractAssets: false"